### DISCUSSION TITLE
World Map Plugin Disable X in top right

### DISCUSSION THREAD

**Discussion by LeoFurnari** [runelite/runelite#17980] [Ideas]
Is it possible to remove the X in the interface for the world map? The clue plugin tooltip sometimes renders below it and the it's hard to navigate to the clue coords with it.

**Comment by SuperNerdEric** [comment]
I too have found this annoying, but I've found that you can right click to select "Focus on Clue Scroll" instead of closing

![image](https://github.com/runelite/runelite/assets/7608429/1b46a0ff-cc95-4d65-bf52-dc9cc68fdea1)
